src/esp/gfx/PbrDrawable.cpp

```diff
@@ -156,6 +159,10 @@ Mn::ResourceKey PbrDrawable::getShaderKey(Mn::UnsignedInt lightCount,
 }
 
 PbrDrawable& PbrDrawable::updateShader() {
+  if (!createRenderer_) {
```

Comment:
How about a comment explaining what's going on here? Why are we early-out returning for this case?